feature-utils/poly-look/src/react-components/progressBar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
If this is simply a move from fbImport, please use `git mv` from now on so that it's flagged as such and differences highlighted.

Reply:
Thanks, I didn't know that command! I'll try to use it from now on 😄